Docs/Operations/station-configuration.md

```diff
@@ -4,17 +4,17 @@ Following the LoRaWAN  Network Server (LNS) specification, each Basics Station (
 
 In the following we describe how to register an LBS in IoT Hub and how to store its configuration.
 
-1. Create an IoT Hub device that has a name equal to the LBS EUI in hex-representation, e.g. `DC-A6-32-FF-FE-B3-2F-C6`.
+1. Create an IoT Hub device that has a name equal to the LBS EUI in hex-representation, e.g. `DCA632FFFEB32FC6`.
```

Comment:
```suggestion
1. Create an IoT Hub device that has a name equal to the LBS EUI in hex-representation, e.g. `DCA632FFFEB32FC6` (case sensitive).
```

Reply:
Not sure it's needed but I discovered it yesterday ;)

Reply:
It shouldn't be case sensitive any longer. Can you still reproduce with the latest version of dev?